.github/scripts/shared/labelable.ts

```diff
@@ -23,6 +23,20 @@ export interface Labelable {
   }[];
 }
 
+// This function tries to find a label on a labelable object (i.e. a pull request or an issue) if present
+export function findLabel(
+  labelable: Labelable,
+  labelToFind: Label,
+): {
+  id: string;
+  name: string;
+} | undefined {
+  // Check if label is present on labelable
+  return labelable?.labels?.find(
```

Comment:
Per the function definition, `labelable` will always be defined . And per the `Labelable` type, it will always have a `labels` field. The optional chaining here shouldn't be needed.

```suggestion
  return labelable.labels.find(
```

Reply:
Fixed in this [commit](https://github.com/MetaMask/metamask-extension/pull/23857/commits/723ca3353fb1c0107d60120ae4e0f69740b87269)